target/linux/x86/config-6.6

```diff
@@ -275,6 +278,9 @@ CONFIG_NET_XGRESS=y
 CONFIG_NLS=y
 # CONFIG_NMI_CHECK_CPU is not set
 # CONFIG_NOHIGHMEM is not set
+CONFIG_NO_HZ=y
+CONFIG_NO_HZ_COMMON=y
+CONFIG_NO_HZ_FULL=y
```

Comment:
I dont think anyone will play with boot parameters. No gain in normal case.
https://jeremyeder.com/2013/11/15/nohz_fullgodmode/